src/server/sourcekites.ts

```diff
@@ -110,7 +112,7 @@ class SourcekiteResponseHandler {
   private handleResponse(data): void {
     this.output += data;
     if (Current.config.isTracingOn) {
-      server.trace("", "" + data);
+      Current.log("SourcekiteResponseHandler", data);
```

Comment:
Minor: now data is logged as a byte array, it's better to keep it as a string as before

Reply:
Thanks! You're right.